scripts/env/kubernetes/sh/045-prepare-s3-containers-configs.sh

```diff
@@ -128,7 +128,7 @@ set_bg_config_param() {
 }
 
 # 'manual' fix until s3_setup is able to handle BG Producer - FIXME
-set_bg_config_param  producer_endpoint  '"http://192.168.134.219:28049"'
+#set_bg_config_param  producer_endpoint  '"http://192.168.134.219:28049"'
```

Comment:
Need to define IO Service with hard-coded IP address, and use that IP here.  Do NOT move this line 131 to bg producer script (remove lines 27 and 28 in script 055).